docs/pages/indexer.mdx

```diff
@@ -0,0 +1,156 @@
+# Indexer
+
+Indexer is an off-chain indexer for on-chain applications build with MUD.
+
+## Why an off-chain indexer?
+
+**Coming soon**
+
+## Installation
+
+These are the steps to install and start Indexer.
+They are written under the assumption you are using `anvil` for your test chain, which is what the getting started package's `pnpm dev` does.
+
+1. Ensure you have these applications installed:
+
+   - [`curl`](https://curl.se/)
+   - [`jq`](https://jqlang.github.io/jq/)
+   - [`node`](https://nodejs.org/en)
+   - [`pnpm`](https://pnpm.io/)
+
+1. Download the source code for MUD.
+
+   ```sh
+   git clone https://github.com/latticexyz/mud.git
+   cd mud
+   ```
+
+1. Remove the `services` package.
+   This package includes various services which are not required for building any of the other packages in the repository.
+
+   ```sh
+   rm -rf packages/services
+   ```
```

Comment:
Does this actually work? @holic mentioned the services package currently includes dependencies for `@latticexyz/network`

Reply:
It took hours, but I figured the exact prereqs and the versions required. So now the directions include `services`.